common/djangoapps/student/views/dashboard.py

```diff
@@ -572,9 +598,12 @@ def student_dashboard(request):
         'EMPTY_DASHBOARD_MESSAGE', None
     )
 
+    course_limit_in_req = request and 'course_limit' in request.GET
+    course_limit = get_dashboard_course_limit() if not course_limit_in_req else None
```

Comment:
So this means that if we set `course_limit` request parameter to anything, it will just list all the courses.  Would it be better to name it something like `ignore_course_limit` or `disable_course_limit` instead to be more clear?